src/utils/cryptoassets.js

```diff
@@ -1,12 +1,13 @@
 import store from '../store'
+import { assets } from '@liquality/cryptoassets'
 
 // Redefine the `cryptoassets` lib to pull from the getter - to include custom tokens
 const cryptoassets = new Proxy({}, {
   get (target, name, receiver) {
-    return Reflect.get(store.getters.cryptoassets, name, receiver)
+    return Reflect.get({ ...store.getters.cryptoassets, ...assets }, name, receiver)
```

Comment:
What is going on here? It looks like you are trying to fix a bug but It's not clear what it is or how to test it. Maybe remove this change from this PR and have another one so its clearer?

Reply:
Yeah, I agree - I will create a new pr for this. 

The problem is that it is getting the tokens from the wallet store, but it do no detect the changes and new token coming from assets so we have to merge both stores to get the new assets.

Reply:
Removed and amended in separate PR